thehive-cortex/app/connectors/cortex/services/ActionOperation.scala

```diff
@@ -135,6 +149,12 @@ class ActionOperationSrv @Inject() (
               _ ← caseSrv.update(caze, Fields.empty.set("customFields", customFields), ModifyConfig(retryOnConflict = 0, version = Some(caze.version)))
             } yield operation.updateStatus(ActionOperationStatus.Success, "")
           }
+        case CloseTask(_, _) ⇒
+          for {
+            initialTask ← findTaskEntity(entity)
+            task ← taskSrv.get(initialTask.id)
```

Comment:
It is not necessary to reload task. Status can be overwritten without checking previous value.